packages/cardano-services/src/Asset/getNftMetadata.ts

```diff
@@ -0,0 +1,32 @@
+import * as Queries from './queries';
+import { Asset, Cardano } from '@cardano-sdk/core';
+import { Logger } from 'ts-log';
+import { MetadataService } from '../Metadata';
+import { Pool, QueryResult } from 'pg';
+
+export interface GetNftMetadataDependencies {
+  metadataService: MetadataService;
+  db: Pool;
+  logger: Logger;
+}
+
+export type AssetPolicyIdAndName = Pick<Asset.AssetInfo, 'name' | 'policyId'>;
+
+export const getNftMetadata = async (
```

Comment:
Perhaps it would be better if there was a `NftMetadataService` interface + `DbSyncNftMetadata` implementation instead of this?